contracts/PetPark.sol

```diff
@@ -1,7 +1,95 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
+import "hardhat/console.sol";
```

Comment:
@shubham-kanodia a few questions:
* does it cost a lot of gas even if no console.log statements are in the code?
* is this high gas cost only during deployment or even during contract executions (when there are no console.log statements)?

Asking because of this info I found on https://hardhat.org/hardhat-network/reference/:
```console.log calls can run in other networks, like mainnet, kovan, ropsten, etc. They do nothing in those networks, but do spend a minimal amount of gas.```

Reply:
It costs gas only during deployment if there are no `console.log` statements. About the deployment cost I think since all the code contained in console.sol is packaged, it would still be a great amount (Its a pretty big [contract](https://github.com/NomicFoundation/hardhat/blob/master/packages/hardhat-core/console.sol))